pkg/mathutils/mathutils.go

```diff
@@ -0,0 +1,9 @@
+package mathutils
+
+// Min returns the smaller of x or y.
+func Min(x, y int64) int64 {
```

Comment:
Please use `math.Min`. No need to implement this logic :)

Reply:
One might think so:D there is only `math.Min` for `float64` in Go.